content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -394,6 +394,43 @@ NAME              PARENTREF
 2001:db8:1:2::a   services/kube-system/kube-dns
 ```
 
+This new feature also allow users to dynamically define the `service-cluster-ip-range` using
+ServiceCIDR objects. During bootstrap, a default ServiceCIDR object named `kubernetes` is created
+from the flags:
+
+```shell
+kubectl get servicecidrs
+```
+```
+NAME         CIDRS         AGE
+kubernetes   10.96.0.0/28  17m
+```
+
+Users can create or delete new ServiceCIDR objects to add or remove ip ranges in order to increase or reduce the number of virtual cluster IPs
+available for their services:
```

Comment:
services -> Services, if you are referring to the API objects.

Reply:
done